plugins/faustwp/includes/replacement/functions.php

```diff
@@ -94,3 +94,20 @@ function normalize_sitemap_entry( $sitemap_entry ) {
 
 	return $sitemap_entry;
 }
+
+/**
+ * Determine what keys within a GraphQL response should have
+ * their values rewritten when rewrites are enabled.
+ *
+ * @return array An array of GraphQL response keys.
+ */
+function graphql_rewrite_keys() {
+	$graphql_rewrite_keys = array( 'url', 'href' );
+
+	/**
+	 * Filter 'faustwp_graphql_rewrite_keys'.
+	 *
+	 * Used to override or extend the response keys targeted to have their values rewritten.
+	 */
+	return apply_filters( 'faustwp_graphql_rewrite_keys', $graphql_rewrite_keys );
```

Comment:
The AC doesn't specify adding a filter. A concern I have with adding this is that we won't have any insights into what else might be breaking or necessary to exclude, as people will just code it up and we'll never know about it.